roles/download/defaults/main.yml

```diff
@@ -65,6 +65,9 @@ docker_image_repo: "docker.io"
 # quay image repo define
 quay_image_repo: "quay.io"
 
+# github image repo define (ex multus only use that)
+github_image_repo: "ghcr.io"
```

Comment:
Please add this `github_image_repo` into inventory/sample/group_vars/all/offline.yml for offline deployment.

At this time, this repo is only for multus, and that doesn't affect to most deployments.
But the repo itself seems so general, so it is nice to consider offline deployment also.

Reply:
Ok will do